File: repos_estudiantes_TARQUI_primera_entrega/RUBIN_DE_CELIS_VALDEZ_ANDRES/INTRODUCCION-POO/EJERCICIO-3/Coche.py
class Coche:
    
    def __init__(self, marca, modelo, velocidad=0):
        self.marca = marca
        self.modelo = modelo
        self.velocidad = velocidad
        
    def acelerar(self):
        self.velocidad += 10
        print(f"{self.marca} {self.modelo} ha acelerado. Velocidad actual: {self.velocidad} km/h")

    def frenar(self):
        self.velocidad -= 5
        if self.velocidad < 0:
            self.velocidad = 0
        print(f"{self.marca} {self.modelo} ha frenado. Velocidad actual: {self.velocidad} km/h")

coche1 = Coche("Toyota", "Corolla")
coche2 = Coche("Ford", "Mustang")
coche1.acelerar()
coche2.acelerar()
coche1.frenar()
coche2.frenar()
print(f"Velocidad final de {coche1.marca} {coche1.modelo}: {coche1.velocidad} km/h")
print(f"Velocidad final de {coche2.marca} {coche2.modelo}: {coche2.velocidad} km/h")

File: repos_estudiantes_ALVAREZ_primera_entrega/GARNICA_FUGUEROA_CAMILA_NATALIA/Introduccion a POO/Ejercicio 2/1. Codigos/src/Ejercicio_2/Coche.java
package Ejercicio_2;

public class Coche {
	public String marca;
	public String modelo;
	public int velocidad;
	
	public Coche(String marca, String modelo, int velocidad) {
		this.marca = marca;
		this.modelo = modelo;
		this.velocidad = velocidad;
	}
	
	public void inicial(){
		System.out.println("La marca es: " + marca);
		System.out.println("El modelo es: " + modelo);
		System.out.println("La velocidad actual es: " + velocidad);
	}

	public void acelerar() {
		velocidad += 10;
		System.out.println("La velocidad actual acelerar es: " + velocidad);
	}
	
	public void frenar() {
		velocidad -= 5;
		System.out.println("La velocidad actuala frenar es: " + velocidad);
	}
}
